training-maquetado/scss/styles.scss

```diff
@@ -1,3 +1,8 @@
-.title {
-  color: red;
+@import url("https://fonts.googleapis.com/css?family=Roboto");
+$text-big: 32px;
```

Comment:
we tend to set this inside a folder call variables in a file `_sizes.scss` you then have to import it in the file you are using it